### PR TITLE
Add new data connector Crowdstrike Falcon Indicators of Compromise

### PR DESCRIPTION
**Pull request by gtheodorio**
Required items, please complete
   
   Change(s):
   - New data connector: CrowdStrike Falcon Indicators of Compromise

   Reason for Change(s):
   - New connector

   Version Updated:
   - No

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Need Help

### CODE REVIEW

**Comment by gtheodorio** [comment]
@microsoft-github-policy-service agree [company="{CrowdStrike}"]

**Comment by gtheodorio** [comment]
@microsoft-github-policy-service agree company="CrowdStrike"

**Comment by v-prasadboke** [comment]
Hello @gtheodorio, Please try to solve KQL validation errors

**Comment by gtheodorio** [comment]
@v-prasadboke validations fixed.

File: Solutions/CrowdStrike Falcon Endpoint Protection/Data Connectors/CrowdstrikeFalconIndicatorsofCompromise/CrowdStrikeFalconIOC_FunctionApp.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    ],
+    "sampleQueries": [
+        {
+            "description": "Threat Intel - Crowdstrike Indicators of Compromise",
+            "query": "ThreatIntelligenceIndicator\n | where SourceSystem == 'CrowdStrike Falcon Indicators of Compromise'\n | sort by TimeGenerated desc"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Indicators of Compromise",
```
Comment (v-prasadboke) [inline Solutions/CrowdStrike Falcon Endpoint Protection/Data Connectors/CrowdstrikeFalconIndicatorsofCompromise/CrowdStrikeFalconIOC_FunctionApp.json:21]: Data type should not have spaces

Comment (gtheodorio) [reply Solutions/CrowdStrike Falcon Endpoint Protection/Data Connectors/CrowdstrikeFalconIndicatorsofCompromise/CrowdStrikeFalconIOC_FunctionApp.json]: Spaces removed

**Comment by v-prasadboke** [comment]
Hello @gtheodorio, Can you share invocation logs ScreenShot

**Review by v-prasadboke — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**